### PR TITLE
Refactoring device heuristics collection to a helper class.

### PR DESCRIPTION
**Pull request by eldhosembabu**
Refactoring device heuristics to a helper class.

### CODE REVIEW

File: FirebaseDynamicLinks/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# v8.6.1
```
Comment (ryanwilson) [inline FirebaseDynamicLinks/CHANGELOG.md:1]: This will likely be 8.7.0

**Review by ryanwilson — APPROVED** [review]
Nit: please fix the changelog otherwise LGTM

**Review by ryanwilson — CHANGES_REQUESTED** [review]
Please update the copyrights in the new files as well.

File: FirebaseDynamicLinks/CHANGELOG.md (hunk 1 above)
Comment (eldhosembabu) [reply FirebaseDynamicLinks/CHANGELOG.md]: Changed!

File: FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Google
```
Comment (ryanwilson) [inline FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.h:2]: Please update to the latest copyright

File: FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.m [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Google
```
Comment (ryanwilson) [inline FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.m:2]: Please update to the latest copyright

File: FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.h (hunk 2 above)
Comment (eldhosembabu) [reply FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.h]: Done

File: FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.m (hunk 3 above)
Comment (eldhosembabu) [reply FirebaseDynamicLinks/Sources/Utilities/FDLDeviceHeuristicsHelper.m]: Done...changed to 2021

**Review by ryanwilson — APPROVED** [review]

**Pull request merged**